common/src/main/java/uk/gov/justice/services/common/log/LoggerConstants.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.justice.services.common.log;
+
+public final class LoggerConstants {
+
+    public static final String REQUEST_DATA = "requestData";
+    public static final String SERVICE_CONTEXT = "serviceContext";
+    public static final String METADATA = "metadata";
+
+    private LoggerConstants() {
```

Comment:
You are correct final abstract is not allowed.  This seems to be the best we can do for reusable constants across multiple classes